poi-4.1.2/pom.xml

```diff
@@ -340,6 +354,12 @@
                                         <exclude>**</exclude>
                                     </excludes>
                                 </filter>
+                                <filter>
+                                    <artifact>org.apache.xmlgraphics:batik-all</artifact>
```

Comment:
No needed if it's a transitive dependency.

Reply:
Removed it. See https://github.com/apache/servicemix-bundles/pull/145#discussion_r485102424